### DISCUSSION TITLE
Input typing responsiveness

### DISCUSSION THREAD

**Discussion by quirm** [sanity-io/sanity#3820] [Sanity Studio Feedback]
Since the early v3 version, typing in an input feels laggy and users in Studio question overall quality because of that :/ Is it a known problem?

Performance snapshot when typing a few characters in input on production build:

<img width="581" alt="image" src="https://user-images.githubusercontent.com/236249/198855265-3fcf83d5-7233-4f8d-87db-1abd8f1e33a3.png">

**Comment by bryandowning** [comment]
I'm seeing the same, but it only became noticeable when I upgraded from `dev-preview.20` to `dev-preview.22`
